protecode/util.py

```diff
@@ -303,7 +304,19 @@ def component_artifact_metadata(
     elif isinstance(artefact.access, cm.S3Access):
         metadata['RESOURCE_TYPE'] = 'application/tar+vm-image-rootfs'
         if not omit_resource_version:
-            metadata['IMAGE_VERSION'] = artefact.version
+            metadata['IMAGE_VERSION'] = resource.version
+
+    elif resource.access.type is cm.AccessType.LOCAL_BLOB:
+        metadata['IMAGE_REFERENCE_NAME'] = resource.name
+        metadata['RESOURCE_TYPE'] = cm.AccessType.LOCAL_BLOB
+        # do not use global access as it is optional
+        img_ref = cnudie.util.oci_artefact_reference(
+            component=f'{component.name}:{component.version}',
+            ocm_repository=component.current_repository_ctx(),
+        )
+        digest = resource.access.localReference
+        metadata['IMAGE_REFERENCE'] = f'{img_ref}:{digest}'
```

Comment:
Use digest reference syntax, i.e. `<image-ref>@sha256:<digest>`

Reply:
also, I removed `current_repository_ctx`. replace w/ current_ocm_repo (caveat: this is a property)